tests/integration/mp-grpc/src/test/java/io/helidon/microprofile/grpc/server/InterceptorsTest.java

```diff
@@ -49,22 +48,16 @@
 /**
  * Functional test for server side interceptors using annotations.
  */
-@ExtendWith(WeldJunit5Extension.class)
+@HelidonTest
 @SuppressWarnings("unchecked")
 public class InterceptorsTest {
 
-    @WeldSetup
-    public WeldInitiator weld = WeldInitiator.of(ServerInterceptorOne.class,
-                                                 ServerInterceptorTwo.class,
-                                                 ServerInterceptorThree.class,
-                                                 ServerInterceptorFour.class,
-                                                 InterceptedServiceOne.class,
-                                                 InterceptedServiceTwo.class,
-                                                 InterceptedServiceThree.class,
-                                                 InterceptedServiceFour.class,
-                                                 InterceptedServiceFive.class,
-                                                 InterceptedServiceSix.class);
+    private CDI weld;
```

Comment:
This deserves to be renamed to `cdi` now

Reply:
Done